### PR TITLE
Updated extract files with tests and updated readme

### PR DESCRIPTION
**Pull request by gawsalya**

### CODE REVIEW

File: live_pipeline/extract_games.py [hunk 1]
```diff
@@ -36,8 +35,8 @@ def parse_game_bs(soup) -> list[str]:
     """Find the user tags for each game."""
     tags = soup.find_all("a", class_="app_tag")
     game_tags = []
-    for each in tags:
-        game_tags.append(each.string.strip())
+    for each_tag in tags:
+        game_tags.append(each_tag.string.strip())
```
Comment (rhKashif) [inline live_pipeline/extract_games.py]: Great improvement for readability!

File: pipeline_games/test_extract_games.py [hunk 2]
```diff
@@ -1 +1,93 @@
...
+from bs4 import BeautifulSoup
+
+from extract_games import get_html, parse_app_id_bs, parse_game_bs, parse_price_bs, system_requirements, get_genre_from_steam, get_developer_name, get_publisher_name, convert_to_csv
+
+
+def test_html_returns_a_string():
+    """Ensure that string is returned for a generic website"""
+    result = get_html("https://www.google.co.uk")
+
+    assert isinstance(result, str) is True
```
Comment (rhKashif) [inline pipeline_games/test_extract_games.py:13]: I usually write: `assert isinstance(result, str) == True` but this looks much nicer!

Comment (gawsalya) [reply pipeline_games/test_extract_games.py:13]: I think pylint like is True but happy to change to == if we're doing that everywhere

**Review by DMcCallion — APPROVED** [review]
Looks good to me

**Comment by gawsalya** [comment]
Thanks for the reviews, both!

**Pull request merged**